tests/benchdnn/brgemm/brgemm.cpp

```diff
@@ -43,11 +43,31 @@ struct dnnl_api_traits<dnnl::impl::cpu::x64::brgemm_kernel_t *> {
         DNN_SAFE_V(dnnl::impl::cpu::x64::brgemm_kernel_destroy(t));
     }
 };
+
+#elif defined(DNNL_AARCH64) && DNNL_AARCH64 == 1 \
+        && DNNL_CPU_RUNTIME != DNNL_RUNTIME_NONE
+template <>
+struct dnnl_api_traits<dnnl::impl::cpu::aarch64::brgemm_kernel_t *> {
+    static void destroy(dnnl::impl::cpu::aarch64::brgemm_kernel_t *t) {
+        DNN_SAFE_V(dnnl::impl::cpu::aarch64::brgemm_kernel_destroy(t));
+    }
+};
 #endif
 
 namespace brgemm {
 
-#if defined(DNNL_X64) && DNNL_X64 == 1 && DNNL_CPU_RUNTIME != DNNL_RUNTIME_NONE
+#if DNNL_CPU_RUNTIME != DNNL_RUNTIME_NONE
+#if defined(DNNL_X64) && DNNL_X64 == 1
+#define brg_x64
+#define namspace_impl dnnl::impl::cpu::x64
```

Comment:
```suggestion
#define namespace_impl dnnl::impl::cpu::x64
```
Here and everywhere.